docs/installation/windows.md

```diff
@@ -16,6 +16,62 @@ There is a shortcut placed in the Start Menu upon installation.
 > Note: On Windows 8.1 / Windows 2012R2, ensure Visual C++ Redistributable 
 > for VS2015 is installed from [here](http://download.microsoft.com/download/9/3/F/93FCF1E7-E6A4-478B-96E7-D4B285925B00/vc_redist.x64.exe):
 
+# Deploying on Nano Server
+
+These instructions assume that Windows PowerShell is running on the Nano Server image and that it has been generated by the [Nano Server Image Builder](https://technet.microsoft.com/en-us/windows-server-docs/get-started/deploy-nano-server).
+Nano Server is a "headless" OS and deployment of PowerShell Core binaries can happen in two different ways:
+
+1. Offline - Mount the Nano Server VHD and unzip the contents of the zip file to your chosen location within the mounted image.
+2. Online - Transfer the zip file over a PowerShell Session and unzip it in your chosen location.
+
+In both cases, you will need the Windows 10 x64 Zip release package and will need to run the commands within an "Administrator" PowerShell instance.
+
+## Offline Deployment of PowerShell Core
+
+1. Use your favorite zip utility to unzip the package to a directory within the mounted Nano Server image.
+2. Unmount the image and boot it
+3. Connect to the inbox instance of Windows PowerShell
+3. Follow the instructions to create a remoting endpoint using the "Executed by another instance of PowerShell on behalf of the instance that it will register" technique and reconnect to the new remoting endpoint on your Nano Server instance. In this case, the "absolute path the the instance's `$PSHOME`" is the directory where it was unzipped.
```

Comment:
> "Executed by another instance of PowerShell on behalf of the instance that it will register" technique

Maybe changing this to the following with a link to the 'Registration' section?
`"Executed by another instance of PowerShell on behalf of the instance that it will register"` [technique](path-to-Registration-section)

> "the "absolute path the the instance's `$PSHOME`" is the directory"

Same comment to this one. Maybe change to
the `"<absolute path to the instance's $PSHOME>"` is the directory ...